app/models/performance_statistics.rb

```diff
@@ -135,4 +165,28 @@ def percentage_of_providers_onboarded
         end
       end
   end
+
+private
+
+  def date_range_query_for_recruitment_cycle_year(cycle_year)
+    start_date = EndOfCycleTimetable::CYCLE_DATES[cycle_year][:apply_reopens]
+
+    query = "created_at >= '#{start_date}'"
+
+    if EndOfCycleTimetable::CYCLE_DATES.to_a.map(&:first).max != cycle_year
```

Comment:
Could this be expressed any more clearly? Are we effectively saying "if we're not looking at the current cycle year"?

Reply:
Yes, I guess just checking for explicitly what we want is better: `if EndOfCycleTimetable::CYCLE_DATES[cycle_year + 1].present?`